src/ai/backend/client/cli/vfolder.py

```diff
@@ -765,3 +765,61 @@ def update_options(name, permission, set_cloneable):
         except Exception as e:
             print_error(e)
             sys.exit(ExitCode.FAILURE)
+
+
+@vfolder.command
+@click.argument("vfolder_id", type=str)
+def shared_vfolder_info(vfolder_id):
+    """Show the vfolder permission information of the given virtual folder.
+
+    \b
+    VFOLDER_ID: ID of a virtual folder.
+    """
+    with Session() as session:
+        try:
+            resp = session.VFolder.shared_vfolder_info(vfolder_id)
+            result = resp.get("shared", [])
+            if len(result) > 0:
+                _result = result[0]
+                print(
+                    'Virtual folder "{0}" (ID: {1})'.format(
+                        _result["vfolder_name"], _result["vfolder_id"]
+                    )
+                )
+                print("- Owner: {0}".format(_result["owner"]))
+                print("- Permission: {0}".format(_result["perm"]))
+                print("- Folder Type: {0}".format(_result["type"]))
+                shared_to = _result.get("shared_to", [])
+                if len(shared_to) > 0:
+                    print("- Shared to:")
+                    for k, v in shared_to.items():
+                        print("\t- {0}: {1}\n".format(k, v))
+        except Exception as e:
+            print_error(e)
+            sys.exit(ExitCode.FAILURE)
+
+
+@vfolder.command()
+@click.argument("vfolder_id", type=str)
+@click.argument("user_id", type=str)
+@click.option(
+    "-p", "--permission", type=str, metavar="PERMISSION", help="Folder's innate permission."
+)
+def update_shared_vfolder(vfolder_id, user_id, permission):
```

Comment:
how about rename it to `update_shared_vf_permission` or something else?
because it only allows to update permission.

Reply:
You're right. Thanks for suggestion!